options.html

```diff
@@ -42,11 +42,14 @@ <h3 class="text-center">EtherAddressLookup</h3>
     <label>Preferred Blockchain Explorer</label>
     <select class="form-control" name="ext-etheraddresslookup-choose_blockchain"
             id="ext-etheraddresslookup-choose_blockchain">
-        <option value="https://etherscan.io/address">Etherscan.io</option>
-        <option value="https://etherchain.org/account">Etherchain.org</option>
+        <option value="https://etherscan.io/address">Etherscan.io *</option>
+        <option value="https://etherchain.org/account">Etherchain.org *</option>
         <option value="https://ethplorer.io/address">Ethplorer.io</option>
         <option value="http://ethergraphs.com/dashboard">Ethergraphs.com</option>
     </select>
+    <div>
+        <p>* ENS address compatible.</p>
+    </div>

```

Comment:
Made this amendment to allow the user to know which explorers will allow them the ability view ENS addresses. We can in future add new functionality to lookup the underlying Ethereum address then link to this on the user's chosen explorer.

Reply:
👍

Reply:
Can we make this text smaller please? 
Just a small UI change, put it in `<small></small>` tags. (I have selective OCD :confused:)

Reply:
I'll make this smaller for now, we can discuss ideas later, though UI isn't really my strong suit.

Reply:
Thanks! Once v1.9 is up, we can arrange a more in-depth discussion on it for later this week when both our schedules are free.